docs/source/api_reference/featurizers.rst

```diff
@@ -251,6 +257,12 @@ RawFeaturizer
   :members:
   :inherited-members:
 
+SNAPFeaturizer
```

Comment:
This looks duplicated? I'm seeing two entries for `SNAPFeaturizer`

Reply:
Indeed. Corrected